ui/src/App.tsx

```diff
@@ -68,43 +73,48 @@ const App = () => {
           style={{ height: 500 }}
         />
       </div>
-      <label>Title</label>
+      <label htmlFor="title">Title</label>
       <input
+        id="title"
         type="text"
         onChange={(e) => {
           setTitle(e.target.value);
         }}
         value={title}
       />
 
-      <label>Start Date</label>
+      <label htmlFor="startDate">Start Date</label>
       <input
+        id="startDate"
         min={formatDate(new Date())}
         type="date"
         onChange={(e) => {
           setStartDate(e.target.value);
         }}
         value={startDate}
       />
-      <label>Start Time</label>
+      <label htmlFor="startTime">Start Time</label>
       <input
+        id="startTime"
         type="time"
         onChange={(e) => {
           setStartTime(e.target.value);
         }}
         value={startTime}
       />
-      <label>End Date</label>
+      <label htmlFor="endDate">End Date</label>
       <input
+        id="endDate"
         min={startDate}
         type="date"
         onChange={(e) => {
           setEndDate(e.target.value);
         }}
         value={endDate}
       />
-      <label>End Time</label>
+      <label htmlFor="endTime">End Time</label>
       <input
+        id="endTime"
```

Comment:
Let's remove this ID - we shouldn't be accessing by Id in tests anyway, prefer to use label text instead.

Reply:
We are not accessing it by its id in the test. This id field is used along with html for attribute on the End Time label to create a relationship between the input and its corresponding label.